.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - '**'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: github.repository == 'GradleUp/shadow'
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-java@v4
+        with:
+          distribution: 'zulu'
+          java-version: 21
+      - uses: gradle/actions/setup-gradle@v3
+        with:
+          cache-read-only: true
+      # Disable CC due to https://github.com/gradle/gradle/issues/22779
+      - run: ./gradlew publish --no-configuration-cache
```

Comment:
If we still want to sync the plugin updates to GradlePluginPortal, need to add `publishPlugins` task here and provide

```sh
GRADLE_PUBLISH_KEY: ${{ secrets.GRADLE_PUBLISH_KEY }}
GRADLE_PUBLISH_SECRET: ${{ secrets.GRADLE_PUBLISH_SECRET }}
```

Reply:
May add GH pages task too, or in the next step with new deployment steps.
It's https://github.com/GradleUp/shadow/blob/main/gradle/ghPages.gradle now, but I'm unfamiliar with GH page deployment, can someone add related tasks or steps?

Reply:
Re: Gradle plugin portal, there's `GRADLE_KEY` and `GRADLE_SECRET` in the org. I think you should be able to see that from https://github.com/GradleUp/shadow/settings/secrets/actions. Let me know if now, we can add you there.

Re: GH pages, they're configured in the settings as well to deploy when pushes are made to `gh-pages`. I would say, maybe try to call `gitPublishCopy` manually?

Reply:
1. https://github.com/GradleUp/shadow/settings/secrets/actions is 404 for me, may cause there is only maintainer permission for.

2. Yeah, seems we can update the env variable and call `gitPublishCopy`

https://github.com/GradleUp/shadow/blob/07f4e63a28b0c61fa9403d2e8320c6c5c0bcc678/gradle/ghPages.gradle#L6

Reply:
@Goooler I have made you an admin, should be much easier now!

![Screenshot 2024-08-02 at 10 17 49](https://github.com/user-attachments/assets/8dd1b7df-204f-4c9d-9c44-d6422bb83768)

Reply:
See it now. Thanks!